lib/modules/commands/give.rb

```diff
@@ -0,0 +1,54 @@
+module DrawBot
+  module DiscordCommands
+    # Gives currency to another user.
+    module Give
```

Comment:
This is basically done, there is a few more cases to handle:

- [ ] Giving to the bot
- [ ] Giving to yourself